### PR TITLE
create a stub for Monticello

### PR DESCRIPTION
**Pull request by MichaelSJohnson**
The purpose of this PR is to add a hometown for Monticello, MN

Help me answer the following:
1. If you have been to Monticello, what did you do?
1. What kind of food do you like?
1. If you have kids, what do they like to do at night?

### CODE REVIEW

**Comment by dave-burke** [comment]
I drive through Monticello on the way to the family cabin. I'd be interested in good places to stop for kids (parks, McDonald's with Play Places, etc.).

**Comment by cheri-tolkinen** [comment]
Yes, I've eaten ice cream in Monticello. :)  I like pizza and hamburgers!

File: michaelsjohnson-monticello.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Monticell, MN
+
+### Restaurants
+Chatters
```
Comment (dave-burke) [inline michaelsjohnson-monticello.md:4]: I'll have to check this place out!

**Comment by MichaelSJohnson** [comment]
> I've eaten ice cream in Monticello
testing reply

**Comment by MichaelSJohnson** [comment]
This pull request resolves #17

**Review by cheri-tolkinen — APPROVED** [review]
Nice job!  Ready to ship

**Pull request merged**